lib/file/getFileBlobAsync.js

```diff
@@ -2,10 +2,23 @@ import { extname } from 'path';
 
 import { ENCODING, FILE_EXTENSION } from './constants';
 
+const wrapBlob = blob => {
+  // Cypress version 5 assigns a function with a compatibility warning
+  // to blob.then, but that makes the Blob actually thenable. We have
+  // to remove that to Promise.resolve not treat it as thenable.
+  if (blob instanceof Cypress.Promise) {
```

Comment:
The eslint configuration forbids direct use of global `Promise` here and this approach appears to work on Cypress 4 in my tests, but we can do an exception if anyone finds an issue with this check.